models/zendesk__ticket_field_history.sql

```diff
@@ -61,14 +61,64 @@ with change_data as (
     from calendar
     left join change_data
         on calendar.ticket_id = change_data.ticket_id
-        and (calendar.date_day >= change_data.valid_from and calendar.date_day < change_data.valid_to)
+        and calendar.date_day = change_data.valid_from
     
     {% if is_incremental() %}
     left join most_recent_data
         on calendar.ticket_id = most_recent_data.ticket_id
-        and (calendar.date_day >= most_recent_data.valid_from and calendar.date_day < most_recent_data.valid_to)
+        and calendar.date_day = most_recent_data.date_day
```

Comment:
I was able to narrow it down that this `and` statement was the cause of the incremental logic failing.